### PR TITLE
Add Documentation for Energenie Pimote

### PR DESCRIPTION
**Pull request by MatejMecka**
## Proposed change

This PR aims to add Documentation for the Parent Pull request.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42599
- Link to parent pull request in the Brands repository: Todo
- This PR fixes or closes issue: N/A

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/energenie_pimote.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Energenie PiMote"
+description: "Control your Energenie devices with a Raspberry Pi and Home Assistant"
+ha_release: "0.118"
+ha_category: Switch
+ha_iot_class: "Assumed State"
+ha_quality_scale: ~
```
Comment (frenck) [inline source/_integrations/energenie_pimote.markdown]:
```suggestion
```

File: source/_integrations/energenie_pimote.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+This integration requires the Pimote addon installed on the Raspberry pi, along with previous setup!
+</div>
+
+### Configuration via the frontend
+
+Make sure you have previously paired them or if you have not follow consult the following [guide](https://www.raspberrypi-spy.co.uk/2017/08/controlling-energenie-power-sockets-with-the-pi-mote-addon/).
+
+In the settings go to `Integrations`, click on the `+` sign to add an integration and click on *Energenie*. After filling out the socket number and name the integration is ready to go!
+
+
```
Comment (frenck) [inline source/_integrations/energenie_pimote.markdown]:
```suggestion
## Configuration

Make sure you have previously paired them or if you have not followed consult the following [guide](https://www.raspberrypi-spy.co.uk/2017/08/controlling-energenie-power-sockets-with-the-pi-mote-addon/).

In the settings go to `Integrations`, click on the `+` sign to add an integration and click on *Energenie*. After filling out the socket number and name the integration is ready to go!

```

File: source/_integrations/energenie_pimote.markdown [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ha_codeowners:
+  - '@matejmecka'
+ha_domain: energenie
+---
+### Energenie
+
+This integration allows users to use their Energenie Pimote devices to be controlled via Home Assistant.
+
+<div class='note'>
+This integration requires the Pimote addon installed on the Raspberry pi, along with previous setup!
```
Comment (frenck) [inline source/_integrations/energenie_pimote.markdown]: previous setup?

File: source/_integrations/energenie_pimote.markdown [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ha_iot_class: "Assumed State"
+ha_quality_scale: ~
+ha_config_flow: true
+ha_codeowners:
+  - '@matejmecka'
+ha_domain: energenie
+---
+### Energenie
+
+This integration allows users to use their Energenie Pimote devices to be controlled via Home Assistant.
```
Comment (frenck) [inline source/_integrations/energenie_pimote.markdown]:
```suggestion
This integration allows you to use and control your Energenie Pimote devices via Home Assistant.
```

File: source/_integrations/energenie_pimote.markdown [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ha_release: "0.118"
+ha_category: Switch
+ha_iot_class: "Assumed State"
+ha_quality_scale: ~
+ha_config_flow: true
+ha_codeowners:
+  - '@matejmecka'
+ha_domain: energenie
+---
+### Energenie
```
Comment (frenck) [inline source/_integrations/energenie_pimote.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @MatejMecka,

I've reviewed your PR and left some comments. Could you please take a look? Thanks 👍

File: source/_integrations/energenie_pimote.markdown (hunk 3 above)
Comment (MatejMecka) [reply source/_integrations/energenie_pimote.markdown]: What do you think is better way to reference it? 

`This integration requires having your switch previously assigned an ID from the link below?`

`This integration requires having followed the integration paired by giving it an ID which you can find below`

Comment (frenck) [reply source/_integrations/energenie_pimote.markdown]: I'm confused by a previous setup as a whole?

What if I don't have a previous setup? Does this mean there is a prerequisite?

Comment (MatejMecka) [reply source/_integrations/energenie_pimote.markdown]: The prerequisite would be assigning an ID to the plug the person owns, that the Energenie addon would know and communicate with

Comment (frenck) [reply source/_integrations/energenie_pimote.markdown]: So, maybe I'm not understanding it or you are missing my point.

If a random new user gets here, never used anything of this before and reads: "along with previous setup" he will magically know what a previous setup is?

Comment (MatejMecka) [reply source/_integrations/energenie_pimote.markdown]: I get it now. i'll fix it

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**